README.rst

```diff
@@ -4,14 +4,14 @@ Equistore
 |test| |docs|
 
 Equistore is a specialized data storage format suited to all your atomistic
-machine learning needs and more. Think of an NumPy "ndarray" or a pytorch "Tensor"
-carrying extra metadata for atomistic systems.
+machine learning needs and more. Think of an NumPy "ndarray" or a pytorch
+"Tensor" carrying extra metadata for atomistic systems.
```

Comment:
It's not actually specific to materials, since we also do molecules & friends. How about "particle systems"? Or be more explicit and use `atomistic and other particles systems`?

Reply:
That'd be great! Just making sure the verbiage is flexible enough to point to what we do 😂